eng/pipelines/templates/stages/archetype-python-release.yml

```diff
@@ -134,6 +134,7 @@ stages:
                           BlobSASKey: '$(azure-sdk-docs-prod-sas)'
                           BlobName: '$(azure-sdk-docs-prod-blob-name)'
                           TargetLanguage: 'python'
+                          ArtifactLocation: '$(Pipeline.Workspace)/${{parameters.ArtifactName}}'
```

Comment:
Shouldn't this be `$(Pipeline.Workspace)/${{artifact.safeName}}`?

Reply:
We put Doc files in `$(Pipeline.Workspace)/${{artifact.safeName}}`, but the source code which we parse the artifact metadata from is inside of `packages` folder, (aka  `ArtifactName`).